app/View/Show.js

```diff
@@ -1,21 +1,76 @@
 import React from 'react';
-import Datagrid from '../Component/Datagrid/Datagrid';
+import Inflector from 'inflected'
+import shouldComponentUpdate from 'omniscient/shouldupdate';
```

Comment:
You need to rebase and use `import { shouldComponentUpdate } from 'react-immutable-render-mixin';`